x-pack/plugins/security/server/routes/views/index.test.ts

```diff
@@ -17,14 +17,15 @@ describe('View routes', () => {
 
     defineViewRoutes(routeParamsMock);
 
-    expect(routeParamsMock.router.get.mock.calls.map(([{ path }]) => path)).toMatchInlineSnapshot(`
+    expect(routeParamsMock.httpResources.register.mock.calls.map(([{ path }]) => path))
+      .toMatchInlineSnapshot(`
       Array [
-        "/security/account",
         "/security/logged_out",
-        "/logout",
-        "/security/overwritten_session",
       ]
     `);
+    expect(routeParamsMock.router.get.mock.calls.map(([{ path }]) => path)).toMatchInlineSnapshot(
+      `Array []`
+    );
```

Comment:
Not sure to understand why some routes are no longer in either assertions?

Reply:
I'm confused by this too

Reply:
Because `/security/account` &  `/security/overwritten_session` use a shorthand method 
`httpResources.registerCoreApp`.
While `/logout ` uses `registerAnonymousCoreApp` helper. I didn't add them since the test case name states only `/login` route check. To note: `routeParamsMock` is called for Login public API endpoints.

Reply:
That shows that we violate one of our development principles: `Prefer one way to do things`  . Lets probably remove `registerCoreApp` & `registerAnonymousCoreApp` shorthands?

Reply:
> That shows that we violate one of our development principles: `Prefer one way to do things` . Lets probably remove `registerCoreApp` & `registerAnonymousCoreApp` shorthands?

I think that makes sense, but I'll defer to your team on that

Reply:
removed